doc/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'FOSHttpCache'
-copyright = '2014, FriendsOfSymfony'
+copyright = u'2014–2016, FriendsOfSymfony'
```

Comment:
Ah, does that work for .rst, too? Our [current docs](http://foshttpcache.readthedocs.io/en/stable/) still have 2014.